tools/check_format.py

```diff
@@ -94,7 +94,7 @@ def fixFileContents(file_path):
   for line in fileinput.input(file_path, inplace=True):
     # Strip double space after '.'  This may prove overenthusiastic and need to
     # be restricted to comments and metadata files but works for now.
-    print "%s" % (line.replace('.  ', '. ').rstrip())
+    sys.stdout.write("%s" % (line.replace('.  ', '. ')))
```

Comment:
Without the rstrip() call won't this eliminate an old behavior where errant spaces at the end of the line were fixed? Or is this handled somewhere else?

Reply:
Oh, I wasn't trying to fix errant spaces at the end of the line when I wrote this.   It wouldn't be a bad thing to keep in, but trailing spaces are also not picked up by check_format so would have the same problem.

If we'd prefer I can add "no trailing spaces and must have terminal newline" to check_format instead of this change.  I think it would require a second regexp pass in check format

Reply:
In that case then I think leave the rstrip out, this LGTM in it's current form. fix_format matching check_format is the most important criteria.